content/en/tracing/profiler/getting_started.md

```diff
@@ -76,70 +76,102 @@ The Datadog Profiler requires [JDK Flight Recorder][1]. The Datadog Profiler lib
 
-     **Note**: Profiler is available in the `ddtrace` library for versions 0.36+.
+**Installation**
 
-3. To automatically profile your code, set `DD_PROFILING_ENABLED` environment variable to `true` when you use `ddtrace-run`:
+Install `ddtrace`, which provides both tracing and profiling functionalities:
 
-    ```
-    DD_PROFILING_ENABLED=true ddtrace-run python app.py
-    ```
-    **Note:** `DD_PROFILING_ENABLED` is only supported in `dd-trace` version 0.40+. Use the alternate method if you are running an older version of `dd-trace`.
+```shell
+pip install ddtrace
+```
 
-    **Alternate method**
+**Note**: Profiling requires the `ddtrace` library version 0.40+.
 
-    If you prefer to instrument the profiler through code, import `ddtrace.profile.auto`. After import, the Profiler starts:
+If you're using a platform where `ddtrace` binary distribution is not
+available, you will need to install a development environment.
 
-    ```python
-    import ddtrace.profiling.auto
-    ```
+For example, on Alpine Linux, this can be done with:
+```shell
+apk install gcc musl-dev linux-headers
+```
 
-4. After a minute or two, visualize your profiles on the [Datadog APM > Profiler page][2].
+**Usage**
 
-- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use environment variables to set the parameters:
+To automatically profile your code, set the `DD_PROFILING_ENABLED` environment
+variable to `true` when you use `ddtrace-run`:
 
-| Environment variable                             | Type          | Description                                                                                      |
-| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
-| `DD_PROFILING_ENABLED`                           | Boolean       | Set to `true` to enable profiler. Supported from tracer version 0.40+.              |
-| `DD_SERVICE`                                     | String        | The Datadog [service][3] name.     |
-| `DD_ENV`                                         | String        | The Datadog [environment][4] name, for example, `production`. |
-| `DD_VERSION`                                     | String        | The version of your application.                             |
-| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.   |
+    DD_PROFILING_ENABLED=true \
+    DD_ENV=prod \
+    DD_SERVICE=my-web-app \
+    DD_VERSION=1.0.3 \
+    dd-trace-run python app.py
 
-<div class="alert alert-info">
-Recommended for advanced usage only.
-</div>
+It is strongly recommended that you add tags like `service` or `version`, as they
```

Comment:
I wonder if folks will think they need to do something in addition to the env vars above. Maybe something like

> In addition to the env, service, and version shown above, you can add other tags like `DD_TAGS=canary:true,shard:3` which will provide the ability to slice and dice...

Or maybe we just mention DD_TAGS and leave details for below. I'm not sure, just throwing something out to make it concrete. Non-blocking.